code-studio/src/js/components/EnrollmentStatus.jsx

```diff
@@ -0,0 +1,87 @@
+/*jshint scripturl:true*/
+
+window.dashboard = window.dashboard || {};
+
+var ModuleSection = React.createClass({
+  propTypes: {
+    moduleType: React.PropTypes.string,
+    status: React.PropTypes.string
+  },
+
+  render: function () {
+    var ribbonClass = 'ribbon ' + this.props.status;
+    return (
+      <div className="module_assignment">
+        {this.props.moduleType}
+        <div className="ribbon-wrapper">
+          <div className={ribbonClass}/>
+        </div>
+      </div>
+    );
+  }
+});
+
+var CourseUnitSection = React.createClass({
+  propTypes: {
+    courseUnit: React.PropTypes.object
```

Comment:
You can also append `.isRequired` to your PropTypes in cases where they should always be there.